snowflake-cortex/prepare_data.sh

```diff
@@ -0,0 +1,26 @@
+
+# create database, schema and table for consumer's example data
+snow sql -q"
+
+CREATE OR REPLACE DATABASE SPOTIFY_RATINGS_DB;
+CREATE OR REPLACE SCHEMA SPOTIFY_RATINGS_SCHEMA;
+
+CREATE OR REPLACE TABLE SPOTIFY_RATINGS_DB.SPOTIFY_RATINGS_SCHEMA.SPOTIFY_CONSUMER_DATA(
+    name VARCHAR;
+    artists VARCHAR;
+    danceabilty FLOAT
+);"
+
+# loading sportify dataset into table stage
+snow object stage copy ./data/spotify_consumer.csv @%SPOTIFY_CONSUMER_DATA --database SPOTIFY_RATINGS_DB --schema SPOTIFY_RATINGS_SCHEMA
```

Comment:
Is the % a typo?

Reply:
If I remember correctly, the % is indicated in the doc to load directly to a table stage, I am going to doublecheck, either way, this file is not being used yet